config/samples/sfnettest/README.md

```diff
@@ -1,35 +1,47 @@
 # Onload application example: sfnettest
 
-Here, we run a small utility, [`sfnt-pingpong` from sfnettest](https://github.com/Xilinx-CNS/cns-sfnettest), in a client and server pair to demonstrate Onload acceleration.
+Here, we run a small utility, [`sfnt-pingpong` from sfnettest](https://github.com/Xilinx-CNS/cns-sfnettest), in a
+client and server pair to demonstrate Onload acceleration.
+
+The sfnettest image is solely focused on its performance utils and thus has a micro shell environment; in-depth network
+inspection should be performed using dedicated software or Onload tools available on the node's host filesystem.
 
 ## Deploy
 
-The example will require customisation to your environment. By default, this will deploy two pods running on nodes named `compute-0` and `compute-1`:
+The following example [client-server.yaml](client-server.yaml) will require customisation to your environment.
+Review the defaults and apply:
+
+* Node names: `compute-0` and `compute-1`
+* [Multus network](../../../docs/nad.md): `ipvlan-bond0`
+* Namespace: `default`
+* Onload accelerated
+* [Onload profile](../../../README.md#using-onload-profiles): `onload-latency-profile`
 
 ```sh
-kubectl apply -f client-server.yaml
+kubectl apply --validate=true -f client-server.yaml
 ```
 
 ## Interactive test
 
-Obtain the SFC interface's IP address of the `onload-sfnettest-server` pod:
+Obtain the SFC interface's IP address of the `onload-sfnettest-server` pod, here `198.19.0.1`:
 
 ```sh
-$ kubectl describe pod onload-sfnettest-server | grep AddedInterface
-  Normal  AddedInterface  24s   multus   Add eth0 [192.168.8.114/23] from openshift-sdn
-  Normal  AddedInterface  24s   multus   Add net1 [198.19.0.1/16] from default/ipvlan-bond0
+$ kubectl get events --field-selector involvedObject.name=sfnettest --field-selector reason=AddedInterface
+LAST SEEN   TYPE     REASON           OBJECT                        MESSAGE
+24s         Normal   AddedInterface   pod/onload-sfnettest-client   Add eth0 [192.168.6.203/23] from openshift-sdn
+24s         Normal   AddedInterface   pod/onload-sfnettest-client   Add net1 [198.19.1.1/16] from default/ipvlan-bond0
+24s         Normal   AddedInterface   pod/onload-sfnettest-server   Add eth0 [192.168.8.143/23] from openshift-sdn
+24s         Normal   AddedInterface   pod/onload-sfnettest-server   Add net1 [198.19.0.1/16] from default/ipvlan-bond0
 ```
 
 The server pod is already running the accelerated `sfnt-pingpong` instance.
 
-Run the client from the `onload-sfnettest-client` pod:
+Run `sfnt-pingpong` as a client within the `onload-sfnettest-client` pod, which has an accelerated environment:
 
 ```sh
 kubectl exec onload-sfnettest-client -- sfnt-pingpong udp 198.19.0.1
 ```
 
-You will likely want to [use an Onload profile](../../../README.md#using-onload-profiles).
```

Comment:
If you are removing this line now that the pods are configured to use the profile by default, I would expand the section above that mentions the profile. I think it is worth emphasizing how to apply profiles and the effect that they can have.